test/integration/host_test.rb

```diff
@@ -267,6 +267,9 @@ def multiple_actions_div
   end
 
   describe "hosts index multiple actions" do
+    after do
+      DatabaseCleaner.clean
```

Comment:
did you add this with the comment, I thought this was only adding assert_equal. Also I think we do DatabaseCleaner.clean already https://github.com/theforeman/foreman/blob/develop/test/test_helper.rb#L328